news/2025-06-11-moving-to-vs2022.md

```diff
@@ -0,0 +1,15 @@
+# Moving to Visual Studio 2022 as default windows compiler
+
+Microsoft's Visual Studio (VS) 2019 compiler has reached its
+[end of life](https://learn.microsoft.com/en-us/lifecycle/products/visual-studio-2019)
+over a year ago. In the meantime, several projects have moved on and
+fail to compile with VS2019.
+
+We are planning to update our default compilers on windows to the (fully compatible)
+successor VS2022 in one week from now.
```

Comment:
is the goal to change "one wekk from from now" to a specific date the second before you press merge?

Reply:
Not really. I like to keep a bit of flexibility, because we're rarely exactly punctual with anything we announce 😅 

But if people prefer exact dates (or even times), fine by me.